sources/packages/api/src/route-controllers/education-program-offering/models/education-program-offering.dto.ts

```diff
@@ -65,7 +165,7 @@ export interface ProgramOfferingDto {
   showYearOfStudy?: boolean;
   hasOfferingWILComponent: string;
   offeringWILType?: string;
-  breaksAndWeeks: StudyBreaksAndWeeks;
+  breaksAndWeeks: StudyBreaksAndWeeksOutDTO;
```

Comment:
pls remove this old DTO interface if its not used

Reply:
The old DTO is part offering entity modal to define contract to the JSON column.

Reply:
if this DTO is used in any controller, please consider renaming it according to our DTO naming convention.